qa/suites/rgw/notifications/tasks/test_first_rabbitmq.yaml

```diff
@@ -0,0 +1,11 @@
+tasks:
```

Comment:
why do you call this file "test_first_rabbitmq.yaml" and not "test_rabbitmq.yaml"

Reply:
Just for the sake that it gets tested before `test_kafka.yaml`. Will change this at the end.